src/store/GirderAPI.ts

```diff
@@ -518,14 +526,23 @@ function parseTiles(items: IGirderItem[], tiles: ITileMeta[]) {
   const zs = new Map<number, Set<number>>();
   const cs = new Set<number>();
 
+  const channelInt = new Map<string | null, number>();
   const lookup = new Map<string, IImage[]>();
   tiles.forEach((tile, i) => {
     const item = items[i]!;
+    const metadata = getNumericMetadata(item.name);
+    if (metadata.chan !== null && !channelInt.has(metadata.chan)) {
+      channelInt.set(metadata.chan, channelInt.size);
+    }
+
     tile.frames.forEach((frame, j) => {
-      const t = +frame.TheT;
-      const xy = +(frame.IndexXY || 0);
-      const z = +(frame.IndexZ !== undefined ? frame.IndexZ : frame.PositionZ);
-      const c = +frame.TheC;
+      const t = metadata.t || +frame.TheT;
```

Comment:
I think we want these to be `metadata.t !== undefined ? metadata.t || +frame.TheT` so that zero behaves properly?

Reply:
See fada1e5.